content/docs/other-guides/virtual-dev/getting-started-multipass.md

```diff
@@ -4,11 +4,8 @@ description = "Quickly get Kubeflow running locally on native hypervisors"
 weight = 2
 +++
 
-{{% alert title="Alpha version" color="warning" %}}
-This feature is currently in <b>Alpha</b> with limited support. The Kubeflow
-team is interested in any feedback you may have, in particular with regards to
-usability of the feature.
-{{% /alert %}}
+{{% alpha-status 
+  feedbacklink="https://github.com/kubeflow/mpi-operator/issues" %}}
```

Comment:
We need to change the feedback link, as the current link points to the issue tracker for another Kubeflow component (MPI operator). I think the best issue tracker for the Multipass/MicroK8s installation is `kubeflow/kubeflow`:

```
feedbacklink="https://github.com/kubeflow/kubeflow/issues"
```

Reply:
Thanks @sarahmaddox for review. I have modified the files as per your review feedback. Please review/approve. Feel free to let me know if further change is needed.